integrationconnectors/endpointattachment/main.tf

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_integration_connectors_endpoint_attachment" "default" {
```

Comment:
Of note, as at time of writing, the tests for this resource didn't run because the other sample failed first; if adding `test.yaml` to the other sample, this one will need further validation.

Reply:
Added skip for the managed zone resource, need one more gcbrun/